src/app/models/domain/hightlight.ts

```diff
@@ -23,5 +23,6 @@ export interface Highlight {
   project: Project;
   startDate?: string;
   endDate?: string;
+  description?: string;
```

Comment:
```suggestion
  description: string;
```
The description should not be nullable